generators/boilerplate/templates/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function add(a, b) {
+  return a + b;
```

Comment:
if we all know it works I honestly don't think we should have this in. It only takes users of the generator time to remove this code and delete the tests. Moot IMHO.